v3/plugins/redhat/vscode-didact/0.1.15/meta.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v2
+publisher: redhat
+name: vscode-didact
+version: 0.1.15
+type: VS Code extension
+displayName: Didact
+title: Didact Tutorial Tools by Red Hat
+description: Provides interactive tutorial support to Visual Studio Code, via didact links calling VS Code commands, Markdown, AsciiDoc, and the VS Code webview.
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/bfitzpat/vsxtemp
+category: Other
+firstPublicationDate: "2020-05-06"
```

Comment:
I think that usually it is close to the first publication of the plugin in the registry. Even if we don't know exactly when it will be merged, maybe we can put the today date.
or using the release on VS Code marketplace, which is 2020-06-18